src/Utils/String.hs

```diff
@@ -0,0 +1,35 @@
+{-|
+Description : Utility functions for displaying strings
+-}
+
+module Utils.String where
+
+import Data.List
+
+-- Prepend l to r and append b
+surrounds :: [a] -> [a] -> [a] -> [a]
+surrounds l r b = l ++ b ++ r
+
+-- | Prepend and append a to b
+surround :: [a] -> [a] -> [a]
+surround a b = surrounds a a b
+
+-- | Insert b between each lr-surrounded element of a list
+surroundMany :: [a] -> [a] -> [a] -> [[a]] -> [a]
+surroundMany l b r ss = ((intercalate b . map (surrounds l r)) ss)
+
+-- | Double quotes a string
+quote :: String -> String
+quote = surround "\""
```

Comment:
However, I do get the feeling that there may be some builtin functions that we can use to the same effect. In the future, if we find that some of these are being used in only one (or none) places, we can probably pull them out.

Reply:
Yep, if we find existing functions we should replace them. I did a quick hoogle search while writing these functions and did not find anything though.